torch/_ops.py

```diff
@@ -251,8 +251,11 @@ def __call__(self, *args, **kwargs):
         dispatch_key_set = _compute_keyset(args, kwargs, self.non_fallthrough_keys)
         return self.dispatch(dispatch_key_set.highestPriorityTypeId(), *args, **kwargs)
 
+    def __str__(self):
+        return f"torch.ops.{self.name()}"
+
```

Comment:
Should just be `{self.name()}`. It's not guaranteed that a HigherOrderOp is accessible under torch.ops